src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1255,6 +1255,10 @@
       <Project>{6e0a079e-85f1-45a1-ad5b-9855e4344809}</Project>
       <Name>Units</Name>
     </ProjectReference>
+    <ProjectReference Include="..\Libraries\PythonNodeModels\PythonNodeModels.csproj">
+      <Project>{8872CA17-C10D-43B9-8393-5C5A57065EB0}</Project>
+      <Name>PythonNodeModels</Name>
```

Comment:
While I don't *think* this causes a circular dependency, it is a bit odd to have DynamoCoreWPF depend on a node library.

 If we could figure out a way to solve this with reflection or some other mechanism - for example an interface in DynamoServiecs that might better decouple the core from our libraries and make it clear what the required interfaces actually are.

Reply:
On the other hand, one could make the argument that this is now a core requirement for a UI feature and as such we need to have this reference.

You may want to spend more time generalizing this if we still intend to allow for fully dynamic engine loading in the future - if not, then maybe defining those interfaces is not worth it at the moment.

Reply:
@mjkkirschner I feel the problem can be traced back to why package manager UI is part of DynamoCoreWpf. To me it seems PM should be more a view extension but not the current way that UI in CoreWpf but data model as extension.. On that note, if the original design is that PM should be part of Core then I feel this dependency is accurate.. Or we could move Python loading to DynamoCore but that also seems not necessary.

Reply:
@QilongTang - there is actually a PackageManagerUIViewExtension - we just never finished moving the UI out of core and into the view extension :) - more tech debt.

Reply:
Ah.. @mjkkirschner Then are you fine of this temp dependency and we shall get to it. Our team have Package Manager UI refresh this quarter and it seems the tech debt can be address in that epic?

Reply:
sounds good - maybe we just make a jira task for this (python references) and another for the packageManager UI refactor tech debt to track both of them?

Reply:
Will do @mjkkirschner

Reply:
Logged as https://jira.autodesk.com/browse/DYN-3765